docs/release-notes/8.0.asciidoc

```diff
@@ -1,6 +1,55 @@
 [[release-notes-header-8.0.0]]
 == 8.0
 
+[discrete]
+[[release-notes-8.0.0]]
+=== 8.0.0
+
+[discrete]
+[[post-upgrade-reqs-8.0.0]]
+==== Post upgrade requirements
+* Changes to the indicator match rule's default threat indicator path (<<rule-ui-advanced-params, *Indicator prefix override*>>) might require you to update existing rules, or create new ones, after upgrading to {stack} version 8.0.0. Be mindful of the following ({pull}118821[#118821]):
```

Comment:
```suggestion
* Changes to the indicator match rule's default threat indicator path (<<rule-ui-advanced-params, *Indicator prefix override*>>) might require you to update existing rules, or create new ones, after upgrading to {stack} version 8.0.0 ({pull}118821[#118821]). Be mindful of the following:
```

Reply:
Commas around "or create new ones" seem unnecessary and break up the sentence's flow.